src/algorithms/signal_source/gnuradio_blocks/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ endif()
 
 
 set(SIGNAL_SOURCE_GR_BLOCKS_SOURCES
+    fifo_reader.cc
```

Comment:
does this need to be conditional on being a Unix build? I don't know enough about Windows, but you say this is for Unix FIFOs

Reply:
It should compile without problems on windows as the c++ code does not contain any OS specific components. 
I have however neither knowledge nor a Windows machine around to test how FIFOs work under Windows.. My research did not leave me confident enough to say if it would work or not. Therefore I documented that I is implemented for a Unix FIFO to prevent confusion. Might work under Windows, but might also require some adaptation. I think it should be safe however to include it in the CMakeLists.txt
(or do you think that could possibly the reason why the windows volkgnsssdr build is failling?)